### DISCUSSION TITLE
🚨 [Projects BETA] Permission Warning - Any organization's member can edit... 🤕😱

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#10447] [Projects and Issues]
🚨🚨🚨
Örgütlenimdeki herhangi üye yetkisizce "Projects Beta" içinde değişiklik yapabiliyor. (_Any member in the organization can make changes in "Projects Beta" without authorization._) Bu üyeler yetki verilmediği halde gizli başlıkları görebiliyor. (_These members can see hidden titles without authorization._)

**Comment by JessicaMulein** [comment]
(Not an answer)
I was actually able to open my public project (base level write) in an incognito window and the UI at least went through the motions of deleting it and removing it from view. Thankfully it seems to still be there upon refresh.
